docs/system-designer/README.md

```diff
@@ -1,43 +1,41 @@
 # System Designer Introduction
 
-The nio System Designer is the graphical user interface used to build your nio system. This overview provides you with a quick tour of the user interface including the navigation, work areas, and toolbars.
+The nio System Designer is the graphical user interface used to build your nio system.
 
+> Take the [nio System Designer Workshop](https://workshops.n.io/system-designer) to go hands-on with the system Designer
 
-{% video %}https://www.youtube.com/watch?v=GFs5EK6zao8{% endvideo %}
-
+---
 ## System Designer Navigation
 
-The screen of the System Designer is laid out in three columns with the systems column and instances and services list on the left; breadcrumb, contextual toolbar, and canvas in the middle; and the block library on the right.
-
-![map of System Designer](/img/system-designer-overview.png)
-
-### Systems Column
+<center>
+<img src="/img/system-designer-overview.png" height="350" vspace="10" />
+</center>
 
-A list of [systems](/systems/README.md) is displayed in a column on the left side of the screen. The designer automatically creates an abbreviation for each system name.
-
-To create a system, click the **`+`** button in the lower-left corner.
-
-To change into the system context and view a system, click the system name.
+### Systems List
+* The designer automatically creates an abbreviation for each system name.
+* To create a system, click the **`+`** button in the lower-left corner.
+* To change into the system context and view a system, click the system name.
+* Learn more about [systems](/systems)
 
 ### Instances and Services List
-
-A list of [instances](/instances/README.md) is displayed in the left navigation panel next to the systems column.
-
-To view the list of [services](/services/README.md) within an instance, click the arrow next to the instance name. The list of services displays below the instance name.
-
-To change into the instance context and view the contents of an instance in the canvas, click the instance name.
-
-To enter the service context and view the contents of a service in the canvas, click a service name.
+* To the right of the systems column.
+* To view the  within an instance, click the arrow next to the instance name.
+    * The list of services displays below the instance name.
+* To view the contents of an instance in the canvas, click the instance name.
+* To enter the service context and view the contents of a service, click a service name.
+* Learn more about [instances](/instances) and [services](/services)
 
 ### Breadcrumb
+* As you navigate from system to service, the breadcrumb allows you to easily identify your location in the system.
 
-As you travel up and down the levels of your system, note that the breadcrumb changes allowing you to easily identify your location in the system.
-
-![breadcrumb hierarchy](/img/cloud/hierarchy.gif)
+<center>
```

Comment:
Do we need `<center>` tags around every image? Maybe we should look into changing the style rules for the theme. @kyliedale @jennyknuth thoughts?

Reply:
I switched to a style tag for each image... as I work through them, I'm getting closer to something we could convert to a class... they're just all pretty different... I recommend slow playing it until everything is 100%, then going down that road.

Reply:
I agree that we certainly don't need every image centered and I like the new look. I am happy to create a class or a custom gitbook block for image placement and standardization. @deliciousmonster I am curious about the deprecated html tags/attributes and if/how best to use them in markdown. Are you using these temporarily or do you find they are the way to go to overcome markdown's limitations?